web/src/components/storage/ConfigureDeviceMenu.tsx

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) [2025] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useRef, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import {
+  MenuToggle,
+  Split,
+  Flex,
+  Label,
+  DrilldownMenu,
+  MenuContent,
+  Divider,
+  MenuContainer,
+  Menu,
+  MenuList,
+  MenuItem,
+} from "@patternfly/react-core";
+import MenuDeviceDescription from "./MenuDeviceDescription";
+import { useAvailableDevices } from "~/queries/storage";
+import { useConfigModel, useModel } from "~/queries/storage/config-model";
+import { deviceLabel } from "~/components/storage/utils";
+import { STORAGE as PATHS } from "~/routes/paths";
+import { sprintf } from "sprintf-js";
+import { _, n_ } from "~/i18n";
+import { StorageDevice } from "~/types/storage";
+
+type DisksDrillDownMenuItemProps = {
+  /** Available devices to be chosen */
+  devices: StorageDevice[];
+  /** The amount of drives already configured */
+  drivesCount: number;
+  /** Callback function to be triggered when a device is selected */
+  onDeviceClick: (deviceName: StorageDevice["name"]) => void;
+};
+
+/**
+ * Internal component holding the logic for rendering the disks drilldown menu
+ */
+const DisksDrillDownMenuItem = ({
+  drivesCount,
+  devices,
+  onDeviceClick,
+}: DisksDrillDownMenuItemProps) => {
+  const isDisabled = !devices.length;
+
+  const disabledDescription = _("Already using all availalbe disks.");
+  const enabledDescription = drivesCount
+    ? sprintf(
+        n_(
+          "Extends the installation beyond the currently selected disk",
+          "Extends the installation beyond the current %d disks",
+          drivesCount,
+        ),
+        drivesCount,
+      )
+    : _("Extends the installation using a disk");
```

Comment:
All those *Extends* should be *Extend*.

Reply:
As discussed int the IRC channel, I do not agree. It's true that for actions we use imperative form, but this is a description of such an action: "**Select** whatever" -> "It **does** that thing"

It's also true that there is an inconsistency with the same description for the "Add LVM" action. But the problem is there, not here.

What I mean is that, **IMHO** we should look at the entire Agama before taking isolated decisions.

Last but not least, I actually think that these descriptions deserves a better review beyond the consistency on the verb form and I fully trust in the fact that we could improve them a bit.

Reply:
Looks like @ancorgs finally decided to [go for your proposal](https://github.com/joseivanlopez/agama/pull/7/commits/3d0d411084217731675a61530bb56ea76d431f6e). I'm going to review the rest of the interface to make it consistent with storage.